README.md

```diff
@@ -33,24 +30,25 @@ These are marked as `peerDependencies` of `stryker-karma-runner` so you get a wa
 
 ## Configuring
 
-You can either configure the karma test runner from the `stryker.conf.js` file or from the command line. This readme describes how to do it via the config file.
+You can either configure the karma test runner from the `stryker.conf.js` file or from the command line. 
+This README describes how to do use the `stryker.conf.js` config file.
 
 ### Load the plugin
 
 In order to use the `stryker-karma-runner` it must me loaded in the stryker mutation testing framework via the stryker configuration. 
-Easiest is to *leave out* the `plugins` section from your config entirely. That way, all node_modules starting with `stryker-` will be loaded.
+Easiest is to *not have a `plugins` section* from your config file. That way, all node_modules starting with `stryker-` will be loaded.

```

Comment:
The easiest way to achieve this, is *not have a `plugins` section* in your config file. That way, all `node_modules` starting with `stryker-` will be loaded.